fe/fe-core/src/main/java/com/starrocks/common/util/Util.java

```diff
@@ -353,7 +353,7 @@ public static String dumpThread(Thread t, int lineNum) {
     //      Base64.encodeBase64String("user:passwd".getBytes());
     // If no auth info, pass a null.
     public static String getResultForUrl(String urlStr, String encodedAuthInfo, int connectTimeoutMs,
-                                         int readTimeoutMs) {
+                                         int readTimeoutMs) throws  Exception {
         StringBuilder sb = new StringBuilder();
         InputStream stream = null;
```

Comment:
why to change the heartbeat timeout to be configurable?

Reply:
not configurable, just same to be/broker heartbeat timeout